server/hooks.go

```diff
@@ -0,0 +1,25 @@
+package main
+
+import (
+	"strings"
+
+	"github.com/mattermost/mattermost-server/model"
+	"github.com/mattermost/mattermost-server/plugin"
+)
+
+func (p *Plugin) ChannelHasBeenCreated(c *plugin.Context, channel *model.Channel) {
+	// Set the description for any DM channels opened between Surveybot and a user
+	if channel.Type != model.CHANNEL_DIRECT {
+		return
+	}
+
+	if !strings.HasPrefix(channel.Name, p.botUserId+"__") && !strings.HasSuffix(channel.Name, "__"+p.botUserId) {
+		return
+	}
+
+	channel.Header = "Surveybot collects user feedback to improve Mattermost. [Learn more](https://mattermost.com/pl/default-nps)."
```

Comment:
Page is not yet available for this URL but I guess it will be to follow.

Reply:
Yep, there's a separate ticket to go set that up once the docs are finished

Reply:
:+1: for using a permalink